rc/extra/just.kak

```diff
@@ -28,25 +28,21 @@ add-highlighter shared/justfile/comment  region '#' '$'  fill comment
 add-highlighter shared/justfile/double_string region '"' (?<!\\)(\\\\)*" fill string
 add-highlighter shared/justfile/single_string region "'" (?<!\\)(\\\\)*' fill string
 add-highlighter shared/justfile/inline   region '`' '`' ref sh
-add-highlighter shared/justfile/shell    region '^\h+' '^[^\h]' group
 
+add-highlighter shared/justfile/shell  region '^\h+' '^[^\h]' group
 add-highlighter shared/justfile/shell/ ref sh
+add-highlighter shared/justfile/shell/ regex '(\{{2})([\w-]+)(\}{2})' 1:operator 2:variable 3:operator
 
 add-highlighter shared/justfile/content/ regex '^(@)?([\w-]+)(?:\s(.+))?\s?(:)(.+)?$' 1:operator 2:function 3:value 4:operator 5:type
 add-highlighter shared/justfile/content/ regex '([=+])' 1:operator
 add-highlighter shared/justfile/content/ regex '^([\w-]+)\s=' 1:value
 
-hook global WinSetOption filetype=justfile %{
-    hook window InsertChar \n -group justfile-indent just-indent-on-new-line
+hook -group justfile-highlight global WinSetOption filetype=justfile %{
     add-highlighter window/justfile ref justfile
-
-    # variable substititution highlighting
-    add-highlighter shared/justfile/shell regex '(\{{2})([\w-]+)(\}{2})' 1:operator 2:variable 3:operator
-
```

Comment:
@TeddyDD I think this used to be incorrect, can you check that it works as expected?

Reply:
This is a hack that highlight `{{variable}}` substitutions in shell regions. I'm not sure if this can be done other way. I'll check it after Wednesday, too busy right now :(

Reply:
I fixed it here https://github.com/mawww/kakoune/pull/2622 Feel free to pull those changes here or I rebase after this PR will be merged.

Reply:
They are the changes I had made as well

Reply:
I only saw part of the diff or I'm tired :| sorry :)